<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Provisioning</class>
 <widget class="QDialog" name="Provisioning">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>504</width>
    <height>590</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Provisioning Editor</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout01">
   <item>
    <widget class="QPlainTextEdit" name="plainTextEdit_main"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_open">
       <property name="text">
        <string>&amp;Open</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_save">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="whatsThis">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Accept the entries, send them to Connman, and close the dialog.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>Sa&amp;ve</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_delete">
       <property name="text">
        <string>&amp;Delete</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_resetpage">
       <property name="whatsThis">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Clear all entries on the current page.&lt;/p&gt;&lt;p&gt;This button will reset every field on the current page to the default value, which generally means nothing in the field.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>Clear &amp;Page</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QToolButton" name="toolButton_whatsthis">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;What's This&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_cancel">
       <property name="whatsThis">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Close the dialog without sending any entries to Connman.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_statusbar"/>
   </item>
  </layout>
  <action name="actionGlobal">
   <property name="text">
    <string>[global]</string>
   </property>
  </action>
  <action name="actionGlobalName">
   <property name="text">
    <string>Name</string>
   </property>
  </action>
  <action name="actionGlobalDescription">
   <property name="text">
    <string>Description</string>
   </property>
  </action>
  <action name="actionService">
   <property name="text">
    <string>[service_*]</string>
   </property>
  </action>
  <action name="actionServiceType">
   <property name="text">
    <string>Type</string>
   </property>
  </action>
  <action name="actionServiceIPv4">
   <property name="text">
    <string>Pv4</string>
   </property>
  </action>
  <action name="actionServiceIPv6">
   <property name="text">
    <string>IPv6</string>
   </property>
  </action>
  <action name="actionServiceMAC">
   <property name="text">
    <string>MAC</string>
   </property>
  </action>
  <action name="actionServiceNameServers">
   <property name="text">
    <string>NameServers</string>
   </property>
  </action>
  <action name="actionServiceTimeServers">
   <property name="text">
    <string>TimeServers</string>
   </property>
  </action>
  <action name="actionServiceSearchDomains">
   <property name="text">
    <string>SearchDomains</string>
   </property>
  </action>
  <action name="actionServiceDomain">
   <property name="text">
    <string>Domain</string>
   </property>
  </action>
  <action name="actionWifi">
   <property name="text">
    <string>[Wifi]</string>
   </property>
  </action>
  <action name="actionWifiName">
   <property name="text">
    <string>Name</string>
   </property>
  </action>
  <action name="actionWifiSSID">
   <property name="text">
    <string>SSID</string>
   </property>
  </action>
  <action name="actionWifiEAP">
   <property name="text">
    <string>EAP</string>
   </property>
  </action>
  <action name="actionWifiCACertFile">
   <property name="text">
    <string>CACertFile</string>
   </property>
  </action>
  <action name="actionWifiClientCertFile">
   <property name="text">
    <string>ClientCertFile</string>
   </property>
  </action>
  <action name="actionWifiPrivateKeyFile">
   <property name="text">
    <string>PrivateKeyFile</string>
   </property>
  </action>
  <action name="actionWifiPrivateKeyPassphrase">
   <property name="text">
    <string>PrivateKeyPassphrase</string>
   </property>
  </action>
  <action name="actionWifiPrivateKeyPassphraseType">
   <property name="text">
    <string>PrivateKeyPassphraseType</string>
   </property>
  </action>
  <action name="actionWifiIdentity">
   <property name="text">
    <string>Identity</string>
   </property>
  </action>
  <action name="actionWifiPhase2">
   <property name="text">
    <string>Phase2</string>
   </property>
  </action>
  <action name="actionWifiPassphrase">
   <property name="text">
    <string>Passphrase</string>
   </property>
  </action>
  <action name="actionWifiSecurity">
   <property name="text">
    <string>Security</string>
   </property>
  </action>
  <action name="actionWifiHidden">
   <property name="text">
    <string>Hidden</string>
   </property>
  </action>
  <action name="actionTemplateEduroamLong">
   <property name="text">
    <string>Eduroam (long)</string>
   </property>
  </action>
  <action name="actionTemplateEduroamShort">
   <property name="text">
    <string>Eduroam (short)</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_cancel</sender>
   <signal>clicked()</signal>
   <receiver>Provisioning</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>351</x>
     <y>737</y>
    </hint>
    <hint type="destinationlabel">
     <x>310</x>
     <y>-23</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
